editor.planx.uk/src/@planx/components/AddressInput/model.ts

```diff
@@ -31,3 +36,12 @@ export const parseAddressInput = (
 
 export const formatAsSingleLineAddress = (address: Address) =>
   Object.values(address).filter(Boolean).join(", ");
+
+export const validationSchema: SchemaOf<AddressInput> =
+  baseNodeDataValidationSchema.concat(
+    object({
+      fn: string().nullable().required(),
```

Comment:
nit: because `fn` has always been required (just browser enforced before) for this component, can we use `string().required()` here without `nullable()` ?